src/test/func/external_pagemap/external_pagemap.cc

```diff
@@ -0,0 +1,30 @@
+#if defined(SNMALLOC_PASS_THROUGH) || defined(_WIN32)
```

Comment:
If you set a `SNMALLOC_NAME_MANGLE` then you can make the test work on Windows.

Reply:
That's not the problem.  The Windows declarations for the standard malloc functions have a bunch of declspecs in them that we're not using and this causes compile failures.

Reply:
If you had set a name mangle then they wouldn't conflict.